gulpfile.babel.js

```diff
@@ -61,8 +61,16 @@ parser.addArgument(['--jsdoc'], {
 });
 
 parser.addArgument(["--jsdoc-private"], {
-    help: "Whether or not jsdoc will document private functions.",
+    help: "Jsdoc will document private functions.",
     type: Boolean,
+    action: 'storeTrue',
+    defaultValue: local_config.jsdoc_private
+});
+
+parser.addArgument(["--no-jsdoc-private"], {
```

Comment:
The setup here is incorrect. The way you have it, it will create a field named `no_jsdoc_private` in `options` which will stand there *besides* `jsdoc_private`. What you need to do is have the flag modify the field named `jsdoc_private` to false. I'm a bit rusty on the details but I vaguely recall you can set a "destination" for an argument. Probably this needs only one additional field in the 2nd argument to `parser.addArgument` that sets the destination.